edudoc/disciplinary-crash-course/disciplinary-crash-course.md

```diff
@@ -0,0 +1,98 @@
+
+## Disciplinary Log and Banned Competitors
+
+* A list of banned competitors can be found in the [Disciplinary log](https://www.worldcubeassociation.org/disciplinary). If a competitor attempts to attend a competition when they are banned from doing so, please contact the WDC.
+* Depending on the degree of severity and type of violation of the Code of Conduct, the WDC may ban a competitor from competing or attending competitions. Such a ban will be communicated with local Delegates when it is issued. If a competitor is banned from competing, but not from attending competitions, they are permitted to attend competitions. They may also fulfill the roles of officials at competitions at the discretion of the WCA Delegate.
+* The WDC will publish reports on remarkable cases they've managed and issue a quarterly digest showcasing smaller resolved cases from the previous quarter. These publications aim to educate the community and WCA staff, promoting transparency. You can find them in the announcement section on the WCA homepage and forum, with a comprehensive list available in the [Disciplinary log](https://www.worldcubeassociation.org/disciplinary).
+
+## Handling Behavioral Issues of Competitors at Competitions 
+
+* Delegates should primarily address minor behavioral issues of competitors at competitions themselves using [2j](https://www.worldcubeassociation.org/regulations/#2j), [2k](https://www.worldcubeassociation.org/regulations/#2k), and [2l](https://www.worldcubeassociation.org/regulations/#2l) of the regulations and applicable sub-points.
+  * If Delegates have not first tried to handle minor behavior issues by giving warnings and disqualifying competitors under regulation 2k, then the WDC is much less likely to apply any disciplinary action. We would like Delegates to attempt to handle minor incidents at a local level first before WDC gets involved. However, the WDC would like Delegates to report noteworthy behavioral issues so the WDC can record them. The WDC is always willing to give advice and take disciplinary action as needed.
+* If any extraordinary cases occur, please describe them in the Delegate Report.
+  * Please be aware the WDC will likely contact you for further information regarding the case.
+  * If an incident is discovered after the competition, this should be added to the Delegate Report, or emailed to the Delegate Report thread.
+
+## {.page-break-before}
```

Comment:
Putting this page break here messes up the formatting for the bullet points, since markdown doesn't understand that the indented bullet point below is a sub-bullet alongside the bullet points above. We should either:

1. Remove this line entirely and let the text naturally flow onto the next page.
2. If you want to keep all of these points on a single page, you can add `{.page-break-before}` to the end of `## Handling Behavioral Issues of Competitors at Competitions`, although this will move the whole section to a new page and you will have a lot of white space on the first page of the doc. 

Either is fine, although I'm preferential to the first option since it's the simplest and doesn't require further fiddling with page breaks every time content is added or removed that messes up the formatting.